index.js

```diff
@@ -0,0 +1,62 @@
+                    </div>
+                    <div id="head-8" class="highlevel-section-div-2 mb-5">
+                        <div><img src="./assets/lecturer4.jpg" alt="lecture 4" /></div>
+                        <div><h4 style="font-weight: 700;">SohYeong Noh</h4>
+                            <p style="font-style: italic; color:#ec5242; font-size: 0.85rem; font-weight: 700;">Director of Art Centre Nabi and a board member of CC Korea</p><hr style="border: 1px solid #d3d3d3; width: 10%;"><p style="font-size: 0.75rem;">As the main venue for new media art production in Korea, Nabi promotes cross-disciplinary collaboration and understanding among science technology, humanities, and the arts.</p></div>
+                    </div>
+                    <div id="head-9" class="highlevel-section-div-2 mb-5">
+                        <div><img src="./assets/lecturer5.jpg" alt="lecture 6" /></div>
+                        <div><h4 style="font-weight: 700;">SohYeong Noh</h4>
+                            <p style="font-style: italic; color:#ec5242; font-size: 0.85rem; font-weight: 700;">Director of Art Centre Nabi and a board member of CC Korea</p><hr style="border: 1px solid #d3d3d3; width: 10%;"><p style="font-size: 0.75rem;">As the main venue for new media art production in Korea, Nabi promotes cross-disciplinary collaboration and understanding among science technology, humanities, and the arts.</p></div>
+                    </div>
+                    <div id="head-10" class="highlevel-section-div-2 mb-5">
+                        <div><img src="./assets/lecturer6.jpg" alt="lecture 6" /></div>
+                        <div><h4 style="font-weight: 700;">SohYeong Noh</h4>
+                            <p style="font-style: italic; color:#ec5242; font-size: 0.85rem; font-weight: 700;">Director of Art Centre Nabi and a board member of CC Korea</p><hr style="border: 1px solid #d3d3d3; width: 10%;"><p style="font-size: 0.75rem;">As the main venue for new media art production in Korea, Nabi promotes cross-disciplinary collaboration and understanding among science technology, humanities, and the arts.</p></div>
+                    </div>
+                </div>
+                <div id="button-3" class="highlevel-section-div mb-5"><button class="p-2" style="width: 100%; background-color: #ffffff; border: 1px solid #d3d3d3; border-radius: 5%;">
+                    MORE <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="#ec5242" class="bi bi-chevron-down" viewBox="0 0 16 16">
+                        <path fill-rule="evenodd" d="M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z"/>
+                      </svg>
+                </button></div>
```

Comment:
- [ ] Good job so far 👍👍 However, as suggested by the reviewer [here](https://github.com/Abdulwhbhussain/Microverse-Capstone-M1/pull/2#discussion_r1173367654) please consider using an array of objects for feature speakers and then use loop to iterate over your speaker's array instead of manually creating each speaker card ⚒
- [ ] Also, please make sure to create a function that iterates through the speakers list on window load and creates HTML elements with the data from each object ✨ 
- [ ] The reviewer has beautifully explained [here](https://github.com/Abdulwhbhussain/Microverse-Capstone-M1/pull/2#discussion_r1173367654) how you can achieve this. Please follow the steps suggested by the previous reviewer. I know you can do that 💪

Reply:
implemented it. 
"A Reviewer Praising Another Reviewer..."

<img src="https://media0.giphy.com/media/tFLueKy1De1r1xEnFP/giphy.gif"/>

Reply:
done.